sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -15,20 +18,23 @@ namespace Azure.Core
 {
     internal class NextLinkOperationImplementation : IOperation
     {
+        internal const string RehydartionTokenVersion = "1.0.0";
         private const string ApiVersionParam = "api-version";
         private static readonly string[] FailureStates = { "failed", "canceled" };
         private static readonly string[] SuccessStates = { "succeeded" };
 
         private readonly HeaderSource _headerSource;
         private readonly Uri _startRequestUri;
         private readonly OperationFinalStateVia _finalStateVia;
-        private readonly RequestMethod _requestMethod;
         private readonly HttpPipeline _pipeline;
         private readonly string? _apiVersion;
 
         private string? _lastKnownLocation;
         private string _nextRequestUri;
 
+        public string? OperationId { get; }
```

Comment:
Curious: Should this be nullable?  Is it ever valid to have an operation without an ID?

Reply:
removed nullable